src/dotnet/UI.Blazor/Components/VideoAttachmentThumbnail/VideoAttachmentThumbnail.razor

```diff
@@ -22,13 +24,15 @@
     [Parameter] public TextEntryAttachment Attachment { get; set; } = null!;
     [Parameter] public string Class { get; set; } = "";
 
-    private string GetUrl()
+    private (string, string) GetUrls()
```

Comment:
You should use `(string OriginalUrl, string ThumbnailUrl)` here, coz it's unclear what they are.